Safety/Src/tim.c

```diff
@@ -444,7 +444,7 @@ void HAL_TIM_Base_MspInit(TIM_HandleTypeDef* tim_baseHandle)
     HAL_GPIO_Init(PPM_GPIO_Port, &GPIO_InitStruct);
 
     /* TIM14 interrupt Init */
-    HAL_NVIC_SetPriority(TIM14_IRQn, 0, 0);
+    HAL_NVIC_SetPriority(TIM14_IRQn, 1, 0);
```

Comment:
Does lowering the priority of the TIM IRQ make it possible to lose PPM data during a SPI IRQ?

Reply:
Its possible but unlikely, and SPI doesn't work at all if you keep them the same priority.

Reply:
Right. Just don't want to get in a state where the Autopilot can lock up the safety controller via SPI. Should spend some time checking this (ex. by injecting faulty SPI protocol).